src/NuGet.Commands/RestoreCommand.cs

```diff
@@ -538,6 +544,29 @@ private bool ResolutionSucceeded(IEnumerable<RestoreTargetGraph> graphs)
                         correctedPackageName: library.Name);
 
                     target.Libraries.Add(targetLibrary);
+
+                    // Log warnings if the target library used the fallback framework
+                    if (warnForImportsOnGraph && !librariesWithWarnings.Contains(library))
```

Comment:
yes, it would, but currently there is only ever one framework for these build integrated projects.